CMakeLists.txt

```
@@ -104,67 +109,75 @@ set(TEST_FILES
     ${TEST_DIR}/TransformTest.cpp
 )
 
-add_executable(Sample
+add_library(PTSD STATIC
     ${SRC_FILES}
-    ${SRC_DIR}/main.cpp
+    ${INCLUDE_FILES}
 )
-target_link_libraries(Sample
+target_link_libraries(PTSD
     SDL2::SDL2main # IDK what happens if we don't link SDL2main
     ${DEPENDENCY_LINK_LIBRARIES}
 )
-target_include_directories(Sample SYSTEM PRIVATE
+target_include_directories(PTSD SYSTEM PRIVATE
     ${DEPENDENCY_INCLUDE_DIRS}
 )
-target_include_directories(Sample PRIVATE
+target_include_directories(PTSD PRIVATE
     ${INCLUDE_DIR}
 )
 if (${PTSD_ENABLE_PCH})
-    target_precompile_headers(Sample PRIVATE
+    target_precompile_headers(PTSD PRIVATE
         include/pch.hpp
     )
 endif()
-target_compile_options(Sample PRIVATE
+target_compile_options(PTSD PRIVATE
     ${TARGET_COMPILE_OPTIONS}
 )
 
+add_executable(Example EXCLUDE_FROM_ALL
+    ${EXAMPLE_FILES}
+)
+target_link_libraries(Example
+    PTSD
+)
+target_include_directories(Example SYSTEM PRIVATE
+    ${INCLUDE_DIR}
+    ${DEPENDENCY_INCLUDE_DIRS}
+)
+target_include_directories(Example PRIVATE
+    ${EXAMPLE_DIR}/include
+)
+
 enable_testing()
 
-add_executable(Tests
+add_executable(Tests EXCLUDE_FROM_ALL
     ${TEST_FILES}
     ${SRC_FILES}
 )
 target_link_libraries(Tests
-    GTest::gtest_main
```

I think you forgot to put this back.
```suggestion
target_link_libraries(Tests
    GTest::gtest_main
```